thieme-german.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>thieme-german.csl</title>
+    <id>http://www.zotero.org/styles/thieme-german</id>
+    <link href="http://www.zotero.org/styles/thieme-german" rel="self"/>
+    <link href="http://www.thieme.de/de/autorenlounge/fuer-zeitschriftenautoren-1789.htm" rel="documentation"/>
```

Comment:
@juliarojahn, a few questions:

* Just to make sure, this style is supposed to follow the format described in https://www.thieme.de/statics/dokumente/thieme/final/de/dokumente/sw_%20autorenlounge/Richtig_Zitieren.pdf, right?
* Instead of keeping "Thieme E-Journals - Vancouver" from the old style, would "Thieme Fachzeitschriften" be a better name as title of this style?
* Do all the journals you listed publish in German? This style is currently set to US English (with `default-locale="en-US"`), so it will use English translation for CSL terms such as "edition" and "in".